src/js/core.js

```diff
@@ -873,6 +873,16 @@
             }
         },
 
+        clearSelection: function (toEnd) {
```

Comment:
It's true that this function is only a wrapper around `collapseToStart()` and `collapseToEnd()` functions, so @nmielnik thinking makes sense. But if I as a non-native speaker would search through documentation, I would be looking for something like clear selection, cancel selection or reset selection. So I would rather stick with `clearSelection` name.